src/utils.py

```diff
@@ -411,3 +408,37 @@ def add_dropped_objects(original, processed):
         if not np.any(np.where((labeled == i) & processed)):
             reconstructed += (labeled == i)
     return reconstructed.astype('uint8')
+
+
+def make_apply_transformer(func, output_name='output', apply_on=None):
+    class StaticApplyTransformer(BaseTransformer):
+        def transform(self, *args, **kwargs):
+            iterator_length = self.check_input(*args, **kwargs)
```

Comment:
@taraspiotr I don't like the naming. check_input can return True or do some assertion but I don't like that it returns length. I would rather have 2 methods `check_input` and `get_length` that we run one ofter the other,

Reply:
@jakubczakon I fully agree, at first, this function was just checking input and then I added length on the spot without rethinking naming.